app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java

```diff
@@ -798,4 +799,24 @@ private void cancelConfigurationTest() {
         }
     }
 
+    private void showAddBagDialog() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(getString(R.string.nav_add));
```

Comment:
There's `setTitle()` overload that accepts resource id, so `getString()` is not necessary here.